src/fluree/db/util.cljc

```diff
@@ -335,7 +338,6 @@
 
 (defn get-graph
   [jsonld]
-  (cond
-    (contains? jsonld :graph)   (:graph jsonld)
-    (contains? jsonld "@graph") (get jsonld "@graph")
-    :else                       jsonld))
+  (if (contains? jsonld "@graph")
+    (get jsonld "@graph")
+    jsonld))
```

Comment:
And `const`s in this file as well.

Reply:
We can't use a const here because of a circular dependency in the const namespace. At least, I'd rather not hold up this pr to resolve the circular dependency. I'll leave a TODO for that.